<launch>
    <!-- define args -->
    <arg name="posture_file"  default="$(find cloth_assist_framework)/files/posture/posture" />
    <arg name="results_file"  default="$(find cloth_assist_framework)/files/posture/posture.yaml" />
    <arg name="calibration_file"  default="$(find cloth_assist_framework)/files/kinect/calibration.yaml" />

    <!-- posture computation from compute_posture.py-->
    <node pkg="cloth_assist_framework" type="posture.py" name="compute_posture" output="screen">
        <param name="posture_file" value="$(arg posture_file)" />
        <param name="results_file" value="$(arg results_file)" />
        <param name="calibration_file" value="$(arg calibration_file)" />
    </node>
</launch>
